include/deal.II/grid/reference_cell.h

```diff
@@ -183,6 +183,77 @@ namespace ReferenceCell
     return Point<dim>(+0.0, +0.0, +0.0);
   }
 
+  /**
+   * Return i-th unit tangential vector (array of dim-1 elements of Point<dim>)
+   * of a face of the reference cell. The vectors are arranged such that the
+   * cross product between the two vectors return the unit normal vector.
+   */
+  template <int dim>
+  inline Tensor<1, dim>
+  unit_tangential_vectors(const Type &       reference_cell,
+                          const unsigned int face_no,
+                          const unsigned int i)
+  {
+    if (reference_cell == get_hypercube(dim))
+      return GeometryInfo<dim>::unit_tangential_vectors[face_no][i];
```

Comment:
I think your goal is to long-term remove `GeometryInfo`, right? If so, how about reversing the order: putting the existing code here and initializing the `GeometryInfo` field by calling this function?

Not sure whether that's easy, but I thought I'd ask.

Reply:
I see your point. But I would keep it for now as it is (since this is the approach I have taken throughout the whole file). I have put this issue onto the TODO list here: https://github.com/dealii/dealii/issues/11352#issue-761425796